Python/generated_cases.c.h

```diff
@@ -1444,6 +1463,8 @@
             PyObject **args;
             PyObject *self;
             PyObject *callable;
+            /* Skip 1 cache entry */
+            /* Skip 2 cache entries */
```

Comment:
Can it print only that second line?

Reply:
The source is `inst(CALL_LIST_APPEND, (unused/1, unused/2, callable, self, args[oparg] -- unused))`
So it is skipping 1 entry, then skipping 2 entries.
We could combine them into `/* Skip 3 cache entries */` but that would lose the structure, and I don't think it is worth the effort.